src/debugger/ios/iOSPlatform.ts

```diff
@@ -37,6 +38,7 @@ export class IOSPlatform implements IAppPlatform {
         this.projectPath = this.runOptions.projectRoot;
         this.simulatorTarget = this.runOptions.target || IOSPlatform.simulatorString;
         this.isSimulator = this.simulatorTarget.toLowerCase() !== IOSPlatform.deviceString;
+        this.iosProjectPath = this.runOptions.iosProjectPath || null;

```

Comment:
I'd initialize this variable with the proper value with something like:
static DEFAULT_IOS_PROJECT_RELATIVE_PATH = "ios";
this.iosProjectPath = path.join(this.projectPath, !ifNullOrUndefined(this.runOptions.iosProjectPath) ? this.runOptions.iosProjectPath : iOSPlatform.DEFAULT_IOS_PROJECT_RELATIVE_PATH);